public-data/ihme_projection_new_york.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Can you add a README to this folder about where this data comes from and how it might updated?

Reply:
Fixed in 
6015065